django/db/models/fields/related.py

```diff
@@ -1314,7 +1314,7 @@ class ForeignObjectRel(object):
 
     # Reverse relations are always nullable (Django can't enforce that a
     # foreign key on the related model points to this model).
-    null = True
+    null = False
```

Comment:
This attribute isn't defined on 1.8. I'm not sure if changing it to `False` is part of Tom's request. We could remove the `self.field.one_to_many` check from the changes above if so.